lib/pages/risky_times_strategies/strategies_text.rb

```diff
@@ -22,17 +22,21 @@ module StrategiesText
       'yourself, and drinking fluids'
     ].freeze
 
+    # Commented out sections below do not run properly.
```

Comment:
@a-ashbeck if we're not gonna use it anymore just delete. we can do a history later if we want to pull it back out